src/components/ListItem.jsx

```diff
@@ -1,5 +1,56 @@
 import './ListItem.css';
+import { updateItem } from '../api';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+const currentTime = new Date();
+
+export function ListItem({ name, items, listToken }) {
+	const { isChecked, id, totalPurchases, dateLastPurchased } = items;
+
+	const currentDate = new Date();
+
+	if (isChecked) {
+		const currentTimeToSec = currentDate.getTime() / 1000;
+		const lastPurchaseToMilSec = dateLastPurchased.seconds * 1000;
+		const timeElapsed = currentTimeToSec - lastPurchaseToMilSec;
```

Comment:
Here you subtract milliseconds from seconds :sweat_smile: . Both times should be either milliseconds or seconds.

Reply:
:open_mouth: :see_no_evil: true!